### ISSUE TITLE
Consider allowing aria-required on checkboxes

### ISSUE THREAD

**Issue by charmarkk** [w3c/aria#921]
During a discussion on the a11y Slack with, the notion of `required` parity with HTML on checkboxes was raised. Though not often used and a bit odd, there are cases where it is beneficial to have an HTML `required` attribute on a checkbox and, as such, would benefit from `aria-required` being allowed in kind.

**Comment by stes-acc** [comment]
We have this for ages in our business frameworks. We need that, too.

**Comment by joanmarie** [comment]
I don't think this is a parity issue. Parity (1.3) is for us to add attributes that don't exist. We have this attribute. We could apply it now. We need consensus. Therefore I'm removing the 1.3 milestone.

**Comment by joanmarie** [comment]
At today's face-to-face we agreed to make this change. I've committed it to master (Editor's Draft). It can land in the Working Draft after we have implementation, tests, authoring, etc. Closing as FIXED.

**Comment by mbgower** [comment]
Just wanted to flag a concern that adding this with no supporting information in the APG risks it being improperly implemented. The only real use case I know of for this is when someone requires a single checkbox to be checked, such as indicating their consent to a statement. 
Obviously if someone applied aria-required to each of a set of checkboxes, it would lead to a problematic interaction.

A larger need is for a _group_ of checkboxes to require at least one checkbox to be chosen. If there was an equivalent to radiogroup for checkboxes, that could address that need.
